Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
```

Comment:
I've updated the requirements.txt since you worked on this. This is the current list:
```
discord.py==1.6.0
flake8==3.9.0
praw==6.2.0
prawcore==1.0.1
pytest==6.2.3
pytest-cov==2.11.1
pytest-mock==3.5.1
redis==3.5.3
```

Reply:
ok